<launch>
    	<node name="cartographer_ros_navigation" pkg="cartographer_ros" type="cartographer_navigation_node" output="screen"/>

        <param name="max_rrt_node_num" type="int" value="1000" />
        <param name="step_to_check_reach_endpoint" type="int" value="50" />
        <param name="distance_threshold_for_adding" type="double" value="10.0" />
        <param name="distance_threshold_for_updating" type="double" value="1.0" />
        <param name="rotation_threshold_for_updating" type="double" value="0.2" />
        <param name="probability_of_choose_endpoint" type="double" value="0.1" />
        <param name="rrt_grow_step" type="double" value="0.2" />
        <param name="rrt_trim_radius" type="double" value="10.0" />
    
</launch>
